lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -700,6 +700,8 @@ handle_dot([$., H | T] = Original, Line, Column, DotInfo, Scope, Tokens) when ?i
         {error, Reason} ->
           {error, Reason, Original, Tokens}
       end;
+    {_NewLine, _NewColumn, [_Part], Rest} ->
+        {error, {Line, Column, "some sort of error message", [H]}, Rest, Tokens};
```

Comment:
If we were okay with this implementation, yes, we would want to change the error message to something useful.

Reply:
The error message should be: "interpolation is not allowed when invoking functions".